net/packet/udp4.go

```diff
@@ -5,8 +5,8 @@
 package packet
 
 // UDPHeader represents an UDP packet header.
```

Comment:
forgot to rename the doc string here.

But name wise: this makes it sounds like there's a "UDPv4" vs "UDPv6", whereas this is really an `IPv4UDPHeader`, but I guess that's a mouthful. So at least document this a bit more while you're updating the doc?

Reply:
I've seen UDP4 and UDP6 in texts before, when the small differences caused by IP family are relevant (e.g. UDP6 checksums are mandatory, UDP4 are optional; and the checksum pseudo-header differs).

rgr on documentation. I'm doing another naming & docs pass once I've made some API changes to use netaddr in more places.